marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/stepDefinition.sjs

```diff
@@ -44,8 +44,8 @@ function getPermissions() {
 }
 
 function getArtifactNode(artifactName, artifactVersion) {
-    const results = cts.search(cts.andQuery([cts.collectionQuery(collections[0]), cts.jsonPropertyValueQuery('name', artifactName)]));
-    return results.toArray().find(artifact => artifact.toObject().name === artifactName);
+    const stepDef = new StepDefinition().getStepDefinition(artifactName);
```

Comment:
This is fine for now, but just an FYI - I'd like to see impl/stepDefinition.sjs go away completely, as it's confusing having two libraries with the same name (same goes for flow.sjs). The main thing it does is fetch modules, which I'd like to move into a separate, non-step-def-specific library. But again, this is fine for now.